pkg/controller/instance/instance_controller.go

```diff
@@ -359,6 +360,21 @@ func createPlan(mgr manager.Manager, planName string, instance *kudov1alpha1.Ins
 		return err
 	}
 	recorder.Event(instance, "Normal", "PlanCreated", fmt.Sprintf("PlanExecution \"%v\" created", planExecution.Name))
+
+	//update the instance with plan
+	active := &instance.Status.ActivePlan
+	log.Printf("PlanExecutionController: Updating active plan of %v", active)
+	active.Name = planExecution.Name
+	active.Kind = planExecution.Kind
+	active.Namespace = planExecution.Namespace
+	active.APIVersion = planExecution.APIVersion
+	active.UID = planExecution.UID
+	log.Printf("PlanExecutionController: Updating active plan to %v", instance.Status.ActivePlan)
```

Comment:
I would merge the logging statement on line 366 and this one into a single log line.